windows/src/unit-tests/package-info/test/test.json

```diff
@@ -0,0 +1,187 @@
+{
+  "system" : {
+    "keymanDeveloperVersion" : "0.0.0.0",
+    "fileVersion" : "7.0"
+  },
+  "options" : {
+    "readmeFile" : "readme.htm",
+    "graphicFile" : "sideimage.bmp"
+  },
+  "startMenu" : {
+    "folder" : "GFF Amharic Keyboard",
+    "addUninstallEntry" : true,
+    "items" : [
+      {
+        "name" : "Welcome",
+        "filename" : "..\\..\\..\\gff\\gff_amh_7\\source\\welcome.htm"
+      },
+      {
+        "name" : "Amharic Keyboard Manual (Amharic)",
+        "filename" : "AmharicTyping-Amharic.pdf"
+      },
+      {
+        "name" : "Amharic Keyboard Manual (English)",
+        "filename" : "AmharicTyping-English.pdf"
+      },
+      {
+        "name" : "OFL (Abyssinica SIL Licence)",
+        "filename" : "OFL.txt"
+      },
+      {
+        "name" : "GPL (WashRa + Free Serif Licence)",
+        "filename" : "GNU-GPLv2.txt"
+      }
+    ]
+  },
+  "info" : {
+    "website" : {
+      "description" : "http://keyboards.ethiopic.org",
+      "url" : "http://keyboards.ethiopic.org"
+    },
+    "version" : {
+      "description" : "8.0.0"
+    },
+    "name" : {
+      "description" : "GFF Amharic Keyboard"
+    },
+    "copyright" : {
+      "description" : "Creative Commons Attribution 3.0"
+    },
+    "author" : {
+      "description" : "The Ge'ez Frontier Foundation",
+      "url" : "mailto:keyboards@ethiopic.org"
+    }
+  },
+  "files" : [
+    {
+      "name" : "C:\\Projects\\keyman\\open\\windows\\src\\unit-tests\\package-info\\Win32\\Debug\\..\\..\\test\\readme.htm",
+      "description" : "File readme.htm"
```

Comment:
Do the tests work if someone isn't using `C:\\Projects\keyman\...` as a root path? 
I see the test.xml and test.ini only use relative paths.

Reply:
Yeah, I'll update the test to use relative paths. Good catch.